library/src/main/java/com/android/ex/chips/RecipientAlternatesAdapter.java

```diff
@@ -67,11 +67,11 @@
     private static final Map<String, String> sCorrectedPhotoUris = new HashMap<String, String>();
 
     public interface RecipientMatchCallback {
-        public void matchesFound(Map<String, RecipientEntry> results);
+        void matchesFound(Map<String, RecipientEntry> results);
```

Comment:
It may not have been right to begin with, but I think that we should avoid changing any methods from `public`. Probably unlikely that anyone would be calling them, but still, I would rather not change the API at all, for a PR like this.

This comment applies for all the places you did this, there were just a few of them